pkg/handler/init_test.go

```diff
@@ -3,10 +3,43 @@ package handler
 import (
 	"fmt"
 	"os"
+	"testing"
+
+	"github.com/kuoss/venti/pkg/mocker"
+	"github.com/kuoss/venti/pkg/mocker/alertmanager"
+	"github.com/kuoss/venti/pkg/model"
+	"github.com/kuoss/venti/pkg/service"
+)
+
+var (
+	services         *service.Services
+	handlers         *Handlers
+	alertmanagerMock *mocker.Server
+	cfg              = &model.Config{
+		Version:    "Unknown",
+		UserConfig: model.UserConfig{},
+		DatasourceConfig: model.DatasourceConfig{
+			Datasources: []model.Datasource{
+				{Type: model.DatasourceTypePrometheus, Name: "prometheus", IsMain: true},
+			},
+		},
+	}
 )
 
-func init() {
-	err := os.Chdir("../..")
+func TestMain(m *testing.M) {
+	setup()
+	code := m.Run()
+	shutdown()
+	os.Exit(code)
+}
+
+func setup() {
+	var err error
+	alertmanagerMock, err = alertmanager.New(0)
+	if err != nil {
+		panic(err)
+	}
+	err = os.Chdir("../..")
```

Comment:
@jmnote 프로님, 이 부분은 상대 참조로 2depth 위의 디렉토리를 가리키게 되는 것 같은데 
절대 참조를 사용하지 않으신 이유가 있을까요?
상대 참조일 경우, 이 디렉토리 경로의 의미를 주석으로라도 표시해두면 유지보수에 도움이 될 것 같습니다.

직접 Comment를 추가하려다 경로의 의미가 모호해 문의 드립니다.

Reply:
반갑습니다. 첫 리뷰 감사합니다.
생각지 못한 부분인데 좋은 지적입니다. 그런데 절대 참조가 가능할지 모르겠어요. ㅎㅎ
현재 위치가 pkg/handler이니까 ../..로 표현하는 게 자연스러울 것 같고요.
개발자마다 git clone하는 위치가 다르니까 테스트할 때 문제가 생기지 않을까요?

Reply:
```suggestion
	err = os.Chdir("../..") // configuration file path
```
네 절대 참조가 어려우면 위와 같은 정도의 주석이어도 좋을 것 같아서 추가 의견 드립니다.

Reply:
좋은 의견입니다~! 다만 저기는 configuration file path가 아니고 project root이고 저기를 기준으로 etc/ 아래에 configuration file이 있습니다. 따라서 '// project root'라는 주석 추가했습니다.

Reply:
아 configuration이 아니라 project root 에 해당하는 path였네요. 내용 확인했습니다.